chain/chain/src/store/mod.rs

```diff
@@ -352,8 +368,8 @@ impl ChainStore {
     /// But we need to implement a more theoretically correct algorithm if shard layouts will change
     /// more often in the future
     /// <https://github.com/near/nearcore/issues/4877>
-    pub fn get_outgoing_receipts_for_shard(
-        &self,
+    pub fn get_outgoing_receipts_for_shard_from_store(
```

Comment:
Nice! Does it make sense to push this in the same helper module we had created last time?

Reply:
Yes. I just don't want to move too much code at once.